white-paper/wp.tex

```diff
@@ -3,7 +3,258 @@
 \usepackage{float}
 \usepackage[bottom]{footmisc}
 \usepackage{bookmark}
-\usepackage{microtype}
+\usepackage{microtype}\documentclass[12pt,oneside]{article}
+                      \usepackage[utf8]{inputenc}
+                      \usepackage{float}
+                      \usepackage[bottom]{footmisc}
+                      \usepackage{bookmark}
+                      \usepackage{microtype}
+                      \usepackage{amsmath}
+                      \usepackage{multicol}
+                      \usepackage{mdframed}
+                      \usepackage{setspace}
+                      \usepackage{pgfplots}
+                      \usepackage{graphicx}
+                      \usepackage{fancyvrb}
+                      \usepackage[absolute]{textpos}\TPGrid{16}{16}
+                      \usepackage{tikz}
+                        \usetikzlibrary{shapes}
+                        \usetikzlibrary{arrows.meta}
+                        \usetikzlibrary{arrows}
+                        \usetikzlibrary{shadows}
+                        \usetikzlibrary{trees}
+                        \usetikzlibrary{fit}
+                        \usetikzlibrary{calc}
+                        \usetikzlibrary{positioning}
+                        \usetikzlibrary{decorations.pathmorphing}
+                      \usepackage{./tikz-uml}
+                      \usepackage{everypage}
+                        \AddEverypageHook{
+                          \begin{textblock}{0.5}[0,0](0,0)
+                            \tikz \node[fill=myred,minimum width=0.5\TPHorizModule,minimum height=16\TPVertModule] {};
+                          \end{textblock}
+                          \begin{textblock}{0.125}[0,0](0.5,0)
+                            \tikz \node[fill=myblack,inner sep=0, minimum width=0.125\TPHorizModule,minimum height=16\TPVertModule] {};
+                          \end{textblock}
+                        }
+                      \usepackage{xcolor}
+                        \definecolor{firebrick}{HTML}{B22222}
+                        \definecolor{myred}{HTML}{CF0A2C}
+                        \definecolor{myblack}{HTML}{232527}
+                      \newcommand\dd[1]{\colorbox{gray!30}{\texttt{#1}}}
+                      \usepackage{hyperref}
```

Comment:
@IlyaMoskva why so many packages added here?

Reply:
Hm.. According to data from Git, they were added by you on Oct-7 in this commit e29e033111bff160e3b4ae18647515aabadf47ec